ydb/docs/en/core/yql/reference/yql-core/syntax/_includes/alter_table.md

```diff
@@ -36,17 +38,63 @@ ALTER TABLE `series` ADD INDEX `title_index` GLOBAL ON (`title`);
 
 You can specify any index parameters from the [`CREATE TABLE`](../create_table#secondary_index) command.
 
-Deleting an index:
+You can also add a secondary index using the {{ ydb-short-name }} CLI [table index](../../../../reference/ydb-cli/commands/secondary_index#add) command.
+
+### Altering an index {#alter-index}
+
+Indexes have type-specific parameters that can be tuned. Global indexes, whether [synchronous]({{ concept_secondary_index }}#sync) or [asynchronous]({{ concept_secondary_index }}#async), are implemented as hidden tables, and their automatic partitioning settings can be adjusted just like [those of regular tables](#additional-alter).
+
+{% note info %}
+
+Currently, specifying secondary index partitioning settings during index creation is not supported in either the [`ALTER TABLE ADD INDEX`](#add-index) or the [`CREATE TABLE INDEX`](../create_table#secondary_index) clauses.
+
+{% endnote %}
+
+```sql
+ALTER TABLE table_name ALTER INDEX index_name SET partitioning_setting_name value;
+ALTER TABLE table_name ALTER INDEX index_name SET (partitioning_setting_name_1 = value_1, ...);
+```
+
+* `table_name`: The name of the table whose index is to be altered.
+
+* `index_name`: The name of the index to be altered.
+
+* `partitioning_setting_name`: The name of the setting to be altered, which should be one of the following:
+    * [AUTO_PARTITIONING_BY_SIZE]({{ concept_table }}#auto_partitioning_by_size)
+    * [AUTO_PARTITIONING_BY_LOAD]({{ concept_table }}#auto_partitioning_by_load)
+    * [AUTO_PARTITIONING_PARTITION_SIZE_MB]({{ concept_table }}#auto_partitioning_partition_size_mb)
+    * [AUTO_PARTITIONING_MIN_PARTITIONS_COUNT]({{ concept_table }}#auto_partitioning_min_partitions_count)
+    * [AUTO_PARTITIONING_MAX_PARTITIONS_COUNT]({{ concept_table }}#auto_partitioning_max_partitions_count)
```

Comment:
```suggestion
* `<table_name>`: The table name whose index is to be modified.

* `<index_name>`: The index name to be modified.

* `<partitioning_setting_name>`: The name of the setting to be altered, which should be one of the following:
    * [AUTO_PARTITIONING_BY_SIZE]({{ concept_table }}#auto_partitioning_by_size)
    * [AUTO_PARTITIONING_BY_LOAD]({{ concept_table }}#auto_partitioning_by_load)
    * [AUTO_PARTITIONING_PARTITION_SIZE_MB]({{ concept_table }}#auto_partitioning_partition_size_mb)
    * [AUTO_PARTITIONING_MIN_PARTITIONS_COUNT]({{ concept_table }}#auto_partitioning_min_partitions_count)
    * [AUTO_PARTITIONING_MAX_PARTITIONS_COUNT]({{ concept_table }}#auto_partitioning_max_partitions_count)
```

Reply:
I have applied the following parts of the commit suggestion:
- <...>
- "altered" -> "modified"

However, I don't like this suggestion:
```diff
-The name of the table whose...
+The table name whose...
```
To quote ChatGPT:

Potential Misinterpretation:
- The second sentence could be (mistakenly) interpreted to mean that the name itself has an index that is going to be modified. This can happen because "table name" could be read as a standalone phrase, and "whose index is to be modified" might be misunderstood to refer to an index associated directly with the "name" rather than the "table."

This suggestion has (in my opinion) the same potential for misinterpretation:
```diff
-The name of the index to be modified.
+The index name to be modified.
```

so I haven't applied it too.